lecture-notes/3-bld.md

```diff
@@ -967,6 +967,8 @@ text assembly instructions into machine code---unreadable bits and bytes. It
 also adds some metadata about what kind of file it is and what functions are
 where. All this together is called an *object file*---a `.o` file.
 
+<!-- TODO(max): Add section diagram for ELF? -->
```

Comment:
Make sure to mention that sections and segments are different things, even though most tooling likes to pretend that segments don't exist.